### ISSUE TITLE
Copyright (C) 2014 MakeStuff (http://makestuff.eu)

### ISSUE THREAD

**Issue by AnttiLukats** [makestuff/lx9#2]
Copyright (C) 2014 MakeStuff (http://makestuff.eu)

it seems makestuff.eu is no longer maintained?

**Comment by makestuff** [comment]
No, I forgot to pay the domain-registration fee, and it lapsed. Also it was getting quite annoying to maintain the website. Is there something you need from there? I still have an archive of the site somewhere.
